app/controllers/scim_rails/scim_service_controller.rb

```diff
@@ -0,0 +1,11 @@
+module ScimRails
+  class ScimServiceController < ScimRails::ApplicationController
+    def configuration
+      ScimRails.config.before_scim_response.call(request.params) unless ScimRails.config.before_scim_response.nil?
+
+      ScimRails.config.after_scim_response.call(ScimRails.config.config_schema, "RETRIEVED") unless ScimRails.config.after_scim_response.nil?
```

Comment:
I'd replace these with checks to see if `before/after_scim_response` are procs/lambdas, instead of not-nil:

```suggestion
      ScimRails.config.before_scim_response.call(request.params) if ScimRails.config.before_scim_response.respond_to?(:call)

      ScimRails.config.after_scim_response.call(ScimRails.config.config_schema, "RETRIEVED") if ScimRails.config.after_scim_response.respond_to?(:call)
```